src/Shared/RedisUtility.cs

```diff
@@ -5,13 +5,24 @@
 
 using System;
 using System.Collections.Generic;
+using System.Configuration;
 using System.IO;
 using System.Runtime.Serialization.Formatters.Binary;
 
 namespace Microsoft.Web.Redis
 {
     internal static class RedisUtility
     {
+        internal static ISerializer Serializer;
+
+        static RedisUtility()
+        {
+            var serializerType = ConfigurationManager.AppSettings["RedisSerializerType"];
```

Comment:
Make "RedisSerializerType" a redis session state and output cache parameter. so it will follow same path as other parameters.